### DISCUSSION TITLE
Feature Request: Exclude Contacts from Export

### DISCUSSION THREAD

**Discussion by rodrialto** [ReagentX/imessage-exporter#389] [Ideas]
First of all, thank you for creating this tool! 
In the latest version, I noticed that filtering the contacts whose messages will be exported is now possible, however, I wondered if adding an option to exclude specific contacts from the export process would also be possible.

Thank you in advance for your time and for considering this suggestion!

**Comment by ReagentX** [comment]
That is not supported at this time, but this does sound like a useful feature to include. A new CLI switch to denote the type of filter and a small change in here to allow both `IN` and `NOT IN` in the statement:

https://github.com/ReagentX/imessage-exporter/blob/e7d5ba0dc7a56222f15d686f5ec2fbbd1ec9ad7f/imessage-database/src/tables/messages/message.rs#L637-L649
